test/templates/page/index.njk

```diff
@@ -0,0 +1,14 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+
+{% block content %}
+    {% set example %}
+    <div style="border: 1px solid">
+        <article class="fd-page">
```

Comment:
I know this has been spec'd this way, but `<article>` seems like the wrong tag to use for a "page".  Almost by definition, shouldn't this class be applied to a `<main>` tag?

From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main:
> The **HTML `<main>` element** represents the dominant content of the `<body>` of a document. The main content area consists of content that is directly related to or expands upon the central topic of a document, or the central functionality of an application.